src/server/game/World/World.cpp

```diff
@@ -1118,6 +1118,7 @@ void World::LoadConfigSettings(bool reload)
     m_int_configs[CONFIG_TALENTS_INSPECTING]           = sConfigMgr->GetIntDefault("TalentsInspecting", 1);
     m_bool_configs[CONFIG_CHAT_FAKE_MESSAGE_PREVENTING] = sConfigMgr->GetBoolDefault("ChatFakeMessagePreventing", false);
     m_int_configs[CONFIG_CHAT_STRICT_LINK_CHECKING_SEVERITY] = sConfigMgr->GetIntDefault("ChatStrictLinkChecking.Severity", 0);
+    m_int_configs[CONFIG_CHAT_STRICT_LINK_CHECKING_CUSTOM_LINKS] = sConfigMgr->GetIntDefault("ChatStrictLinkChecking.CustomLinks", 0);
```

Comment:
Maybe rename this option to CONFIG_CHAT_STRICT_LINK_CHECKING_COLOR_LINKS? Because with this option you will allow only color tags, not names or something else changes.

Reply:
that's because the "something else changes" have not been implemented yet, like support to https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/worldserver/worldserver.conf.dist#L3245